analytics/cdk/emr-eks/bin/emr-eks.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import EmrEksStack from '../lib/emr-eks-stack';
+import { Stack } from 'aws-cdk-lib';
+import { EmrEksTeamProps } from '../lib/teams/emrEksTeam';
+import { PolicyStatement } from 'aws-cdk-lib/aws-iam';
+
+const app = new cdk.App();
+
+const account = '123456789012';
+const region = 'eu-west-1';
+
+const executionRolePolicyStatement: PolicyStatement[] = [
+  new PolicyStatement({
+    resources: ['*'],
```

Comment:
Overly wide permissions. The corresponding TF pattern's permissions are here: https://github.com/awslabs/data-on-eks/blob/main/analytics/emr-eks-amp-amg/data.tf#L13